torchbenchmark/__init__.py

```diff
@@ -62,14 +62,16 @@ def _install_deps(model_path: str, verbose: bool = True) -> Tuple[bool, Any]:
     ]
     run_env = os.environ.copy()
     run_env["PYTHONPATH"] = this_dir.parent
+    for key, item in run_env.items():
+        run_env[key] = str(run_env[key])
     run_kwargs = {
         "cwd": model_path,
-        "check": True,
+        "check": 'True',
```

Comment:
"check" is not an environment variable, why it needs to be converted to string as well?